server/src/main/java/org/gluu/oxtrust/api/RegistrationManagementService.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * oxAuth is available under the MIT License (2008). See http://opensource.org/licenses/MIT for full text.
+ *
+ * Copyright (c) 2014, Gluu
+ */
+
+package org.gluu.oxtrust.api;
+
+import com.wordnik.swagger.annotations.ApiOperation;
+import com.wordnik.swagger.annotations.ApiResponse;
+import com.wordnik.swagger.annotations.ApiResponses;
+import org.gluu.oxtrust.api.client.RegistrationManagementRequest;
+import org.gluu.oxtrust.api.client.RegistrationManagementResponse;
+import org.gluu.oxtrust.ldap.service.AttributeService;
+import org.gluu.oxtrust.ldap.service.JsonConfigurationService;
+import org.gluu.oxtrust.ldap.service.OrganizationService;
+import org.gluu.oxtrust.model.GluuOrganization;
+import org.gluu.oxtrust.model.RegistrationConfiguration;
+import org.gluu.oxtrust.service.filter.ProtectedApi;
+import org.gluu.oxtrust.util.OxTrustApiConstants;
+import org.gluu.oxtrust.util.OxTrustConstants;
+import org.slf4j.Logger;
+import org.xdi.config.oxtrust.AppConfiguration;
+import org.xdi.model.GluuAttribute;
+import org.xdi.util.StringHelper;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Implementation for register REST web services.
```

Comment:
Same as above :).
You may want to keep the author, but it's irrelevant since Git stores the author (It's up to you)